.github/workflows/release-operator.yaml

```
@@ -0,0 +1,30 @@
+name: Release Operator
+
+on:
+  pull_request:
+    types:
+    - closed
+    branches:
+      - 'release-v[0-9]+.[0-9]+'
```

Was thinking that since we are targeting this workflow to be run once the candidate release merge into the release branch, we could do the following:

```suggestion
on:
  push:
    branches:
      - 'release-v[0-9]+.[0-9]+'
```

`push` since we'll be only merging once the candidate PR is approved, no?
Thus the conditional on line 13 won't be needed

---

I reckon that the advantage would be that the work is not triggered when a PR is closed without merging, so no conditional needed in the job :)